modules/lm-coursier/src/main/scala/lmcoursier/internal/ResolutionRun.scala

```diff
@@ -87,10 +90,10 @@ object ResolutionRun {
         .withResolutionParams(
           params
             .params
-            .addForceVersion((if (isCompileConfig) params.interProjectDependencies.map(_.moduleVersion) else Nil): _*)
-            .withForceScalaVersion(isCompileOrScalaToolConfig && params.autoScalaLibOpt.nonEmpty)
+            .addForceVersion((if (isSandboxConfig) Nil else params.interProjectDependencies.map(_.moduleVersion)): _*)
+            .withForceScalaVersion(params.autoScalaLibOpt.nonEmpty)
             .withScalaVersionOpt(params.autoScalaLibOpt.map(_._2))
-            .withTypelevel(params.params.typelevel && isCompileOrScalaToolConfig)
+            .withTypelevel(params.params.typelevel)
```

Comment:
Same as above: restricting the configuration that do that may have been useful in some contexts, but it seems that doesn't break tests here…